### PR TITLE
Minor improvements to the release process

### PR DESCRIPTION
**Pull request by PRabahy**
Instruct people to "git fetch" so that if this is their 2nd+ gitian build they will have a fresh bitcoin repo.
Instruct people to add all the known pgp keys to their keyring so that gverify will print more useful info.

### CODE REVIEW

File: doc/release-process.md [hunk 1]
```diff
@@ -41,7 +41,12 @@ Check out the source code in the following directory hierarchy.
 	pushd ./bitcoin
 	export SIGNER=(your Gitian key, ie bluematt, sipa, etc)
 	export VERSION=(new version, e.g. 0.8.0)
+	git fetch
 	git checkout v${VERSION}
+
+  Add other gitian builders keys to your gpg keyring
+
+	gpg --import ./contrib/gitian-downloader/*.pgp
```
Comment (jonasschnelli) [inline doc/release-process.md]: I think the verification/import keys part should go to a section below... maybe a new one.

**Comment by jonasschnelli** [comment]
Nice.
utACK

**Comment by fanquake** [comment]
utACK

**Comment by MarcoFalke** [comment]
utACK

File: doc/release-process.md [hunk 2]
```diff
@@ -83,25 +84,16 @@ NOTE: Offline builds must use the --url flag to ensure Gitian fetches only from
 ```
 The gbuild invocations below <b>DO NOT DO THIS</b> by default.
 
-###Build (and optionally verify) Bitcoin Core for Linux, Windows, and OS X:
+###Build and Sign Bitcoin Core for Linux, Windows, and OS X:
 
 	./bin/gbuild --commit bitcoin=v${VERSION} ../bitcoin/contrib/gitian-descriptors/gitian-linux.yml
 	./bin/gsign --signer $SIGNER --release ${VERSION}-linux --destination ../gitian.sigs/ ../bitcoin/contrib/gitian-descriptors/gitian-linux.yml
-	./bin/gverify -v -d ../gitian.sigs/ -r ${VERSION}-linux ../bitcoin/contrib/gitian-descriptors/gitian-linux.yml
-	mv build/out/bitcoin-*.tar.gz build/out/src/bitcoin-*.tar.gz ../
```
Comment (MarcoFalke) [inline doc/release-process.md]: #7465

**Pull request merged**